project/CelebornBuild.scala

```diff
@@ -895,6 +906,8 @@ object MRClientProjects {
       .settings(
         commonSettings,
         libraryDependencies ++= Seq(
+          Dependencies.commonsCompress,
+          Dependencies.commonsLogging,
```

Comment:
https://www.scala-sbt.org/release/docs/Library-Management.html#Conflict+Management

> By default, the latest revision is selected

Reply:
I had a discussion with @pan3793 about how to pin dependency versions, and we ultimately chose the `dependencyOverrides` approach. see https://github.com/apache/incubator-celeborn/pull/1831#discussion_r1303760048. cc @pan3793